andela/ah-backend-poseidon
# #162162999 Send a verification link to a user via email upon successful registration.

## What does this PR do
- A user receives email notification on successful registration.
- A user verifies new account created via clicking a link provided in the email.

## Description of Task to be completed?
- When a user signs up, he should receive an email verification link.
- When the user clicks on the link provided, he should be redirected to verify account page.

##  How should this be manually tested?
Register a new user on postman by adding a JSON object with these parameters to the api/users/ endpoint.
Please use the format below.
```{
  "user": {
        "email": "user@user.com",
        "username": "user",
        "password": "user1234",
  }
}
```

Go to your email and verify your account by clicking the link
You should see a successfully verified message

## Relevant pivotal tracker stories?
#162162999
